assets/src/stories-editor/components/block-navigation/index.js

```diff
@@ -18,7 +18,7 @@ import BlockNavigationItem from './item';
 import { ALLOWED_MOVABLE_BLOCKS } from '../../constants';
 import './edit.css';
 
-function BlockNavigationList( { blocks,	selectedBlockClientId, selectBlock, callToActionBlock } ) {
+function BlockNavigationList( { blocks, selectedBlockClientId, selectBlock, unMovableBlock } ) {
```

Comment:
Should `unMovableBlock` be a helper function? Seems like it might be useful else where in the code..

Reply:
Can you perhaps elaborate what you had in mind?

Worth noting that it's not really a `boolean` value, but the result of `const unMovableBlock = blocks.find( ( { name } ) => ! ALLOWED_MOVABLE_BLOCKS.includes( name ) );` in the `withSelect` below.

Reply:
Couldn't we make this a reuable function like this. 

`const unMovableBlock = ( name ) => => ! ALLOWED_MOVABLE_BLOCKS.includes( name ) ;`